docs/example-scenario/apps/app-monitoring.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Web Application Monitoring on Azure
+description: This example scenario is relevant to organizations that want to monitor their application hosted in Azure App Service.
+author: adamboeglin
+ms.date: 09/12/2018
+---
+
+# Web Application Monitoring on Azure
+
+Platform-as-a-service (PaaS) offerings on Azure manage compute resources for you and in some ways change how you monitor deployments. Azure includes multiple monitoring services, each of which performs a specific role. Together, these services deliver a comprehensive solution for collecting, analyzing, and acting on telemetry from your applications and the Azure resources they use.
+
+This scenario addresses the monitoring services you can use and describes a dataflow model for use with multiple data sources. When it comes to monitoring, many tools and services work with Azure deployments. In this scenario, we choose readily available services precisely because they are easy to consume. Other monitoring options are discussed later in this article.
+
+## Potential use cases
+
+Consider this scenario for the following use cases:
+
+- Instrumenting a web application for monitoring telemetry.
+- Collecting front-end and back-end telemetry for an application deployed on Azure.
+- Monitoring metrics and quotas associated with services on Azure.
+
+## Architecture
+
+![App Monitor Architecture Diagram][architecture]
+
+This scenario uses a managed Azure environment to host an application and data tier. The data flows through the scenario as follows:
+
+1. User interacts with app.
+1. Browser and App Service emit telemetry.
+1. Azure Application Insights collects and analyzes application health, performance, and usage data.
+1. Developers and administrators can review health, performance, and usage information.
+1. SQL Database emits telemetry.
+1. Azure Monitor collects and analyzes infrastructure metrics and quotas.
+1. Log Analytics collects and analyzes logs and metrics.
+1. Developers and administrators can review health, performance, and usage information.
+
+### Components
+
+- [Azure App Service][Azure App Service] is a PaaS service for building and hosting apps in managed virtual machines. The underlying compute infrastructures on which your apps run is managed for you. App Service provides monitoring of resource usage quotas and app metrics, logging of diagnostic information, and alerts based on metrics. Even better, you can use Application Insights to create [availability tests][availability tests] for testing your application from different regions.
+- [Application Insights][Application Insights] is an extensible Application Performance Management (APM) service for developers and supports multiple platforms. It monitors the application, detects application anomalies such as poor performance and failures, and sends telemetry to the Azure portal.
```

Comment:
Should call out that App Insights is also used for:
- Application logging (exceptions, trace statements)
- [Distributed tracing and correlation](https://docs.microsoft.com/en-us/azure/application-insights/application-insights-correlation)